spec/spec_helper.cr

```diff
@@ -1,3 +1,5 @@
+{% raise("Please use `make spec` or `bin/crystal` when running specs, or set the i_know_what_im_doing flag if you know what you're doing") unless env("CRYSTAL_HAS_WRAPPER") || flag?("i_know_what_im_doing") %}
```

Comment:
It may be better to redirect users to the contributing guide instead of the answer directly.